shopify_app.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.metadata['allowed_push_host'] = 'https://rubygems.org'
 
   s.add_runtime_dependency('browser_sniffer', '~> 1.2.2')
-  s.add_runtime_dependency('rails', '> 5.2.1', '< 6.1')
+  s.add_runtime_dependency('rails', '> 5.2.1', '~> 6.1')
```

Comment:
Is this followed by a `bundle update` the right way to do it?

Reply:
Not answering the `bundler update` question, but is that range valid? I think they are `AND`ed together. So this would prevent Rails 5 from working?

Reply:
Thanks for clarifying! I was definitely confused whether this was an `AND` or a range

Reply:
re:

> I wonder if we can just change this requirement to ~> 6.1 now that we know it'll be safe until v7.

cc @paulomarg

I don't believe this'll be a valid range, so maybe `'< 7.0'`?

Reply:
Oh, this was hidden in the current files, sorry! It seems a little strange that we previously supported `> 5.2.1` rather than `~>5.0` to begin with, but I guess we don't have to drop support for v5 if it still works. So yeah, let's go with the upper cap just to make sure we're not flipping over to breaking changes.